ipatool

```diff
@@ -928,7 +928,10 @@ def backport(ctx, backport_branches, repo, pr):
 
     patches = list(ctx.get_patches())
     os.chdir(cleanpath(ctx.config['clean-repo-path']))
-    github_login = github3.login(token=ctx.config['gh-token']).me().login
+    try:
+        github_login = github3.login(token=ctx.config['gh-token']).me().login
+    except KeyError as exc:
```

Comment:
unused variable exc, should the exception be at least printed?

Reply:
Will do.